packages/ember-data/lib/system/model/states.js

```diff
@@ -685,6 +681,7 @@ var RootState = {
       isDirty: false,
 
       setup: function(internalModel) {
+        internalModel.clearRelationships();
```

Comment:
Should I remove the `internalModel.clearRelationships();` from the `unloadRecord` methods?